apps/api/lib/api_web/controllers/events_controller.ex

```diff
@@ -0,0 +1,30 @@
+defmodule ApiWeb.EventsController do
+  @moduledoc """
+  Web interface for viewing events
+  """
+
+  use ApiWeb, :controller
+
+  alias Core.EventRepo
+
+  action_fallback(ApiWeb.ErrorsController)
+
+  def index(conn, _opts) do
+    {:ok, events} = EventRepo.events()
+    json(conn, %{data: %{events: events}})
+  end
+
+  def show(conn, %{"id" => raw_id}) do
+    with {:ok, id} <- Helpers.Integer.parse(raw_id),
+         {:ok, events} <- EventRepo.get_event(id) do
```

Comment:
I think it's confusing that `EventRepo.get_event` returns a list of events. I think in this case (since we have a separate endpoint) it should return a single event object.